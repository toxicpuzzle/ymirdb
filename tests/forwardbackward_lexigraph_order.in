SET a 1 2 3 
SET b 1 2 3 
SET c 1 2 3
SET d 12 3
SET e 39 0
SET f 102 3
PUSH a b f e
PUSH f d c
PUSH d c
PUSH b c
FORWARD a 
BACKWARD c
bye