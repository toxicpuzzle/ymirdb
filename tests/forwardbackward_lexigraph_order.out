> ok

> ok

> ok

> ok

> ok

> ok

> ok

> ok

> ok

> ok

> b c d e f 

> a b d f 

> bye